src/main/java/com/conveyal/r5/streets/EdgeStore.java

```diff
@@ -1113,6 +1145,30 @@ public void setWalkTimeFactor (double walkTimeFactor) {
         public void setBikeTimeFactor (double bikeTimeFactor) {
             edgeTraversalTimes.setBikeTimeFactor(edgeIndex, bikeTimeFactor);
         }
+
+        public Map<String, Object> attributesForDisplay () {
+            Map<String, Object> map = new HashMap<>();
+            map.put("id", edgeIndex); // Map UI component seems to group edges by this ID to span tile boundaries.
+            map.put("streetClass", (int) streetClasses.get(pairIndex)); // Serialization cannot handle Byte, cast it.
+            // map.put("osmId", getOSMID());
+            map.put("speedKph", getSpeedKph());
+            map.put("lengthM", getLengthM());
+            // FIXME we should employ a method like com.conveyal.r5.labeling.LevelOfTrafficStressLabeler.ltsToInt for this
+            int lts =  getFlag(EdgeFlag.BIKE_LTS_1) ? 1 :
+                        getFlag(EdgeFlag.BIKE_LTS_2) ? 2 :
+                        getFlag(EdgeFlag.BIKE_LTS_3) ? 3 : 4;
+            map.put("lts", lts);
+            map.put("pedestrian", getFlag(EdgeFlag.ALLOWS_PEDESTRIAN));
+            map.put("bike", getFlag(EdgeFlag.ALLOWS_BIKE));
+            map.put("car", getFlag(EdgeFlag.ALLOWS_CAR));
+            if (costFields != null) {
```

Comment:
Here, it would also be helpful to display the `perceivedLengthMultipler` values for walk and bike from the `edgeTraversalTimes`.

Even if we move away from the legacy LADOT generalized costs, other users have set their own custom impedance values for walking and biking (e.g. via #653), and they might want to inspect them.

Reply:
I added a `walkTimeFactor` and `bikeTimeFactor` to the attributes for display when the `edgeTraversalTimes` are present on the `EdgeStore`.